tests/solvers/linear/univariate_test.py

```diff
@@ -163,3 +168,31 @@ def univ_dist(x, y, a, b):
     expected = univ_dist(x, y, a, b + v_b) - univ_dist(x, y, a, b - v_b)
     actual = 2.0 * jnp.vdot(v_b, grad_b)
     np.testing.assert_allclose(actual, expected, rtol=tol, atol=tol)
+
+  @pytest.mark.fast()
+  def test_dual_vectors(self):
+    """The OTT solver outputs meaningful dual variables."""
+    _ = pytest.importorskip("lineax")  # only tested using lineax
+    n = self.n
```

Comment:
Personal pet peeve: don't think it's that necessary to have `n = self.n` or the `x, a, ... = self.x, self.a`...